packages/0xcert-scaffold/src/core/gateway.ts

```diff
@@ -28,7 +28,7 @@ export interface GatewayBase {
    * @param order Order data.
```

Comment:
Why u change chmod here and on other files?

Reply:
Good catch, had some problems setting up env and was changing permissions. This slipped trough. Will fix.